### ISSUE TITLE
Timeout commands

### ISSUE THREAD

**Issue by GiuseppeLaurenza** [radare/radare2-r2pipe#36]
Hi,
I need to set a timeout to the aaaa command, because I need to automatically analyze various samples, so if there is one that can "stuck" the analysis for long time, I prefer to stop and use a lighter command (like "aa")
How I can do it?
Thanks in advance

**Comment by radare** [comment]
There’s anal.timeout eval var iirc. Not sure if its honored . Sending a sigint aka ^c to the process can work too

Let me know

**Comment by GiuseppeLaurenza** [comment]
Can you explain with more details the anal.timeout suggestion?
The signal way is not so easy to apply in my tool

**Comment by GiuseppeLaurenza** [comment]
I try to set anal.timeout=10 to test, I see Timeout message, but than r2 remains stucked

**Comment by radare** [comment]
Yeah its poorly tested. Not sure if theres a timeout for rcons too. Not in laptop right now.
